run-warc-tests.sh

```diff
@@ -32,27 +38,27 @@ isTopLevel"
 trap 'kill $(jobs -pr)' SIGINT SIGTERM
 
 # wayback and proxychains are much happier if they're run from the directory with their config files
-cd ${ARCHIVE_DIR}
+cd "${SCRIPT_DIR}"
 
 # Write the CSV column names
 echo "Creating output file ${OUTPUT}"
-echo ${CSV_COLUMNS} > ${OUTPUT}
+echo "${CSV_COLUMNS}" > "${OUTPUT}"
 
 # Read the archives from the ARCHIVE file
 while IFS=: read ARCHIVE URL; do
 
     # Start the wayback server in tx1he background
     echo ""
     echo "Testing ${URL} from archive ${ARCHIVE}"
-    wayback --proxy ${ARCHIVE} --port ${PORT} > /dev/null 2>&1 &
+    wayback --proxy "${ARCHIVE}" --port "${PORT}" > /dev/null 2>&1 &
 
     # Wait for the server to start up
     while ! PID=$(lsof -Pi :${PORT} -t); do sleep 1; done
 
     # Run a proxified servo on the URL, and save any WARC lines in the output file
-    timeout 2m proxychains ${SERVO} ${URL} | grep WARC | sed -e "s/WARC/${ARCHIVE}/" >> ${OUTPUT}
+    timeout 2m proxychains "${SERVO}" "${URL}" | grep WARC | sed -e "s/WARC/${ARCHIVE}/" >> "${OUTPUT}"
```

Comment:
`$SERVO` is currently a single, whitespace separated string, so quoting it will break this execution. I'd recommend turning the SERVO variable into an array, and expanding it here with `"${SERVO[@]}"`. Might also be nice to rename it to `SERVO_CMD` (I originally thought it was the path to servo itself).

Reply:
Done.